### ISSUE TITLE
Lobotomizer proc undertuned?

### ISSUE THREAD

**Issue by dorkyo** [elysium-project/server#2169]
In some recent testing I believe The Lobotomizer proc "Brain Damage" is doing about 50% of its correct damage. The tooltip states a 200-300 damage proc, yet mine are consistently hitting in the 120-150 range, with crits ~250. It also seems the proc is extremely rare, having a less than 1% chance to happen? If anyone can prove or disprove these theories that would be much appreciated.

**Comment by strudelkuchen** [comment]
http://db.vanillagaming.org/?spell=24388 says its Damage School is Physical which means armor should be taken into account. 

You can test it with friends and tell them to strip naked.

**Comment by dorkyo** [comment]
You're absolutely right, which is why my testing was done on extremely low/no armor targets :D

**Comment by GuybrushGit** [comment]
Works fine in dev
